resources/scripts/ttx_diff.py

```diff
@@ -233,7 +240,53 @@ def stat_like_fontmake(ttx):
     # So until such time as we start writing format 4 axis value tables it doesn't matter
     ver.attrib["value"] = "0x00010001"
 
-def reduce_diff_noise(fontc, fontmake):
+
+def allow_some_off_by_ones(
+    build_dir, fontc, fontmake, container, name_attr, coord_holder
+):
+    fontmake_num_coords = len(fontmake.xpath(f"//{container}/{coord_holder}"))
+    off_by_one_budget = int(FLAGS.off_by_one_budget / 100.0 * fontmake_num_coords)
+    spent = 0
+    if off_by_one_budget == 0:
+        return
+
+    coord_tag = coord_holder.rpartition("/")[-1]
+
+    for fontmake_container in fontmake.xpath(f"//{container}"):
+        name = fontmake_container.attrib[name_attr]
+        fontc_container = select_one(fontc, f"//{container}[@{name_attr}='{name}']")
+
+        fontc_it = fontc_container.iter()
+        for fontmake_el in fontmake_container.iter():
+            fontc_el = next(fontc_it)
```

Comment:
you could `zip` the two iterators. Also I think you'd get StopIteration error here if next() fails to yield because fontmake_container is longer than fontc_container

Reply:
Oops, good catch.